numpy/core/__init__.py

```diff
@@ -11,7 +11,18 @@
     if envkey not in os.environ:
         os.environ[envkey] = '1'
         env_added.append(envkey)
-from . import multiarray
+
+try:
+    from . import multiarray
+except ImportError:
+    msg = """
```

Comment:
``textwrap.dedent`` does that, I'm not aware of something fancier. but yes, may be useful especially for longer strings.